tests/SearchTest.php

```diff
@@ -25,10 +28,16 @@ public static function setUpBeforeClass(): void
             ['id' => 4,    'title' => 'Harry Potter and the Half-Blood Prince', 'comment' => 'The best book'],
             ['id' => 42,   'title' => 'The Hitchhiker\'s Guide to the Galaxy'],
         ];
-        static::$index->addOrUpdateDocuments($documents);
+        static::$index->updateDocuments($documents);
         usleep(10 * 1000);
```

Comment:
Is it difficult to wait for update with a loop on getOneUpdate(updateId)?

Reply:
Yes, it's an alternative to the `usleep` to do a sync method of each method for the test with this kind of loop. But now it would add me a looooooot of work to do it in all the SDKs, because the PHP SDK is not the only one concerned.

Reply:
You can also create only one method who take a update_id and do a loop. like that: 
```php
class Index extends HTTPRequest
{
    public function waitUpdateId($update_id)
    {
        while (1) {
            $res = $this->getUpdateStatus($update_id);
            if ($res['status'] == 'processed' || $res['status'] == 'failed') {
                return $res;
            }
            usleep(500);
        }
    }
}
```

I can propose a PR if you want!

Reply:
Ok as you want :)
Can you do it after we merged this one? It would be clearer to review :)

Reply:
yes!